Kickstarter-iOS/Views/Controllers/LiveStreamContainerMoreMenuViewController.swift

```diff
@@ -0,0 +1,102 @@
+import KsApi
+import Library
+import LiveStream
+import Prelude
+import ReactiveSwift
+import Result
+import UIKit
+
+internal protocol LiveStreamContainerMoreMenuViewControllerDelegate: class {
+  func moreMenuViewControllerWillDismiss(controller: LiveStreamContainerMoreMenuViewController)
+  func moreMenuViewControllerDidShare(controller: LiveStreamContainerMoreMenuViewController,
+                                      liveStreamEvent: LiveStreamEvent)
+  func moreMenuViewControllerDidSetChatHidden(controller: LiveStreamContainerMoreMenuViewController,
+                                              hidden: Bool)
+}
+
+//swiftlint:disable:next type_name
+internal final class LiveStreamContainerMoreMenuViewController: UITableViewController {
+  private let dataSource = LiveStreamContainerMoreMenuDataSource()
+  private let viewModel: LiveStreamContainerMoreMenuViewModelType = LiveStreamContainerMoreMenuViewModel()
+
+  private weak var delegate: LiveStreamContainerMoreMenuViewControllerDelegate?
+
+  internal static func configuredWith(liveStreamEvent: LiveStreamEvent,
+                                      delegate: LiveStreamContainerMoreMenuViewControllerDelegate,
+                                      chatHidden: Bool) -> LiveStreamContainerMoreMenuViewController {
+    let vc = Storyboard.LiveStream.instantiate(LiveStreamContainerMoreMenuViewController.self)
+    vc.viewModel.inputs.configureWith(liveStreamEvent: liveStreamEvent, chatHidden: chatHidden)
+    vc.delegate = delegate
+
+    return vc
+  }
+
+  internal override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.tableView.dataSource = self.dataSource
+    self.tableView.isScrollEnabled = false
+    self.automaticallyAdjustsScrollViewInsets = false
+
+    let emptyFooterView = UIView(frame: CGRect())
+    self.tableView.tableFooterView = emptyFooterView
```

Comment:
Empty footer just ensures that we won't see any trailing separators at the bottom of the table view, but seeing as it's not a scrollable table view I'll remove it.